Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/UI/Ride.hs

```diff
@@ -97,16 +99,18 @@ listDriverRides driverId mbLimit mbOffset mbOnlyActive mbRideStatus = do
     rideDetail <- runInReplica $ QRD.findById ride.id >>= fromMaybeM (VehicleNotFound driverId.getId)
     rideRating <- runInReplica $ QR.findRatingForRide ride.id
     driverNumber <- RD.getDriverNumber rideDetail
-    pure $ mkDriverRideRes rideDetail driverNumber rideRating (ride, booking)
+    mbExophone <- runInReplica $ QExophone.findByPrimaryPhone booking.primaryExophone
```

Comment:
What we should send if exophone was removed from db (via dashboard or manually)?
Now I send `booking.primaryExophone`, but not sure this is correct